src/org/usfirst/frc2175/commandmapper/JoystickEventMapper.java

```diff
@@ -27,6 +29,14 @@ public JoystickEventMapper(RobotConfig robotConfig,
         retractCatapult
                 .whenPressed(new RetractCatapultCommand(robotSubsystems));
 
+        JoystickButton runIntakeIn = gamepadConfig.getIntakeInButton();
+        runIntakeIn.whenPressed(new RunIntakeInGroup(robotSubsystems,
+                robotConfig.getIntakeConfig()));
```

Comment:
Rather than doing `robotConfig.getIntakeConfig` here, pull intakeConfig out to its own variable up above. Like on line 22 of this file!

Like this: `IntakeConfig intakeConfig = robotConfig.getIntakeConfig();`

You can put that on line 23.